lib/util/stream_utils.js

```diff
@@ -1626,3 +1645,8 @@ shaka.util.StreamUtils.minAvifImage_ = 'data:image/avif;base64,AAAAIGZ0eXBhdm' +
 shaka.util.StreamUtils.minImage_ = new Map()
     .set('image/webp', shaka.util.StreamUtils.minWebPImage_)
     .set('image/avif', shaka.util.StreamUtils.minAvifImage_);
+
+/**
+ * @private {!Map.<string, boolean>}
+ */
+shaka.util.StreamUtils.mediaSourceTypeSupportMap = new Map();
```

Comment:
Nit: Since this is private, the name should end with an underscore. "mediaSourceTypeSupportMap_".

Reply:
Understood